pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -38,6 +46,12 @@ export default Vue.extend<Data, Methods, Computed, never>({
   },
 
   beforeMount() {
+    ipcRenderer.on('snapshot', (_, event) => {
+      this.snapshotEvent = event;
+    });
+    if (this.$route.params) {
+      this.snapshotEvent = this.$route.params.event as any;
```

Comment:
We would need to restructure here if we choose to take on changes to route params so that they more closely adhere to `Dictionary<string>`.

```suggestion
   const { type, result, name } = this.$route.params as SnapshotEvent;

   this.snapshotEvent = {
     type, result, name,
   };
```

Reply:
Destructuring here breaks banner functionality, since the `snapshotEvent` must also be `null` in case of no event. I'll revert it.